.github/actions/build-package/action.yml

```diff
@@ -48,7 +48,7 @@ runs:
 
     - name: npm install
       shell: bash
-      run: npm i
+      run: npm i --force
```

Comment:
Currently the documenter package does not make use of a compatible combination of dependencies. Unfortunately it can't just be upgraded easily and almost requires a complete rewrite of the package. The `--force` flag is temporary for now to allow this package to build.